.copywrite.hcl

```
@@ -0,0 +1,10 @@
+schema_version = 1
+
+project {
+  license        = "MPL-2.0"
```

```suggestion
  license = "MPL-2.0"
```
If only https://github.com/hashicorp/vscode-hcl/issues/98 was a thing 😄